src/api-specification/contribution-controller/assembling-a-contribution.md

```diff
@@ -45,7 +45,9 @@
 
 ::: info
 
-The `source` field is optional and is the same as the `sourcePeerId` value when [retrieving contributions](retrieving-contributions.md). 
+The `source` field is optional and is the same as the `sourcePeerId` value when [retrieving contributions](retrieving-contributions.md).
```

Comment:
There have been some changes. 
1. The "source" has been renamed to "**sourcePeerId.**" Therefore, it is necessary to update this attribute in other parts of the document as well.
2. This part is no longer relevant: "The 'source' field is optional and is the same as the 'sourcePeerId'..."